backend/fleet_management/serializers.py

```diff
@@ -95,6 +96,15 @@ def create(self, validated_data):
                 )
             return drive
 
+    def is_valid(self, raise_exception=False):
+        try:
+            super().is_valid(raise_exception=raise_exception)
+        except ValidationError as err:
+            err_codes = err.get_codes()
+            if "non_field_errors" in err_codes and "unique" in err_codes["non_field_errors"]:
+                err.status_code = status.HTTP_409_CONFLICT
```

Comment:
Looking good!
If you are changing status code here, please change it in frontent also.
https://github.com/CodeForPoznan/pah-fm/blob/develop/frontend/src/store/actions.js#L99

Reply:
> If you are changing status code here, please change it in frontent also.

Isn't it done already?

Reply:
👍